src/kOS/Persistence/PersistenceExtensions.cs

```diff
@@ -128,7 +128,7 @@ public static ConfigNode ToConfigNode(this HarddiskFile file, string nodeName)
             {
                 if (SafeHouse.Config.UseCompressedPersistence)
                 {
-                    node.AddValue("binary", EncodeBase64(content.String));
+                    node.AddValue("binary", PersistenceUtilities.EncodeBase64(content.Bytes));
```

Comment:
This class's local EncodeBase64() method tried to encode the string you give it, by mapping it to a byte array using a vanilla ASCII interpretation (see deleted code below in this PR diff output), which will mangle anything that needs to use the higher 8 bits of the 16 bit Unicode char.

This was unnecessary as well, since ``content``, being of type ``FileContent``, already has its own logic to do its own UTF-8 encoding, built-in.  ``content.String`` is the unicode string in chars, while ``content.Bytes`` is the UTF-8 encoded byte array version of that string, and ``FileContent`` does the work to ensure the bytes are UTF-8 rather than just a vanilla dump of the unicode 2-byte patterns.  

Once I decided to use ``content.Bytes`` instead of ``content.String``, that also made the entire reason for ``EncodeBase64()`` go away (it was *just* to encode the string for you, which ``FileContent`` already does, better.)